runtime/seq.go

```diff
@@ -173,6 +173,26 @@ func seqFindFirst(f *Frame, iterable *Object, pred func(*Object) (bool, *BaseExc
 	return false, nil
 }
 
+func seqFindElem(f *Frame, elems []*Object, o *Object) (int, *BaseException) {
+	for i, elem := range elems {
+		var raised *BaseException
+		var eq *Object
+		var found bool
+		eq, raised = Eq(f, elem, o)
```

Comment:
You can remove the eq and raised declarations above and then assign them this way:

```
eq, raised := Eq(f, elem, o)
```

Ditto for found, below.